src/types/api.types.ts

```diff
@@ -0,0 +1,7 @@
+export type LoadingState = {
+  loading: boolean;
+};
+
+export type ErrorState = {
+  error: string | undefined | null;
+};
```

Comment:
We always try and postfix our types with "Def", and interfaces with "Props". Not the end of the world, but if you have the time, could you please change it?
```suggestion
export type LoadingStateDef = {
  loading: boolean;
};

export type ErrorStateDef = {
  error: string | undefined | null;
};
```

Reply:
sure, let me change it.

Reply:
done